src/main/io/serial_cli.c

```diff
@@ -415,7 +415,11 @@ static const char * const lookupTableGPSSBASMode[] = {
 #endif
 
 static const char * const lookupTableCurrentSensor[] = {
-    "NONE", "ADC", "VIRTUAL"
+    "NONE", "ADC", "VIRTUAL", "ESC"
+};
+
+static const char * const lookupTableBatterySensor[] = {
+    "NONE", "ADC", "ESC"
```

Comment:
`NONE` should not be in `lookupTableBatterySensor`, since it is not a valid option. Currently, if it is selected, the code for `ADC` will be executed.